README.md

```diff
@@ -316,6 +316,12 @@ var Gdax = require('gdax');
 var websocket = new Gdax.WebsocketClient(['BTC-USD', 'ETH-USD']);
 websocket.on('message', function(data) { console.log(data); });
 ```
+
+Optionally set the heartbeat mode.
+```javascript
+var websocket = new Gdax.WebsocketClient(['BTC-USD','ETH-USD'], {heartbeat: true});
```

Comment:
Here it would be good to expand upon the optional parameters (websocket URI, authentication) just so that people know more without having to dig into the code themselves to figure out how it works, or why their old implementation is breaking if they've upgraded.

Reply:
ok, I'll update